pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -1068,6 +1069,13 @@ private void setupRouteGenericErrorHandlers() {
             response.status(401);
             response.body(exception.getMessage());
         });
+        exception(DuplicateEntityException.class, (e, request, response) -> {
+            // Tell the user to retry with a different name.  Not logged as an error,
+            // since this can happen during normal operations.
+            response.status(422);
+            response.body(String.format("%s %s is already taken.  Please retry with a different value.",
```

Comment:
does it make sense to change the last word from "value" to `e.getEntityName()`, 
"Please try with a different filter name" for example